tutorpicasso/patches/openedx-dockerfile-pre-assets

```
@@ -1,11 +1,19 @@
+{% set valid_branches = ['teak'] %}
```

@luisfelipec95 why do we need to explicitly list valid branches here?

We should keep in mind that once Tutor releases a new version, the `teak` branch will be removed. I’m asking to avoid needing to come back and constantly update this list of valid branches with every new release.

Personally, I’d prefer to let the build try to run and simply fail if something’s misconfigured on that branch, rather than hardcoding which branches are allowed.

That said, I do think we should clearly document in this plugin what’s required for a branch to be compatible (e.g., plugins index setup or any external steps that were needed to make it work with the `teak` branch).

Also, I’m wondering if this would work with the `main` branch (formerly `nightly`). That’s the branch we should expect to always work. If something extra is needed to make it compatible with `main`, I don’t think that should block this PR, but it’d be good to know what’s missing for full support so we can maybe implement it later.

---

@luisfelipec95 I see that remove the branch validation. What happens now when we use a branch that is "not" valid?. Also, please remember what I mentioned in my comment about updating this plugin's doc and mentioning what needs to be done for a branch in order for the build to be successful as it is with the `teak` branch